content/articles/compile-your-node.js-application-into-a-.exe-file/index.md

```diff
@@ -0,0 +1,556 @@
+
+This causes the need for a simple executable file that can run on any operating system without requiring the user to perform other steps to run the application.
+
+In this tutorial, we'll learn how to create executable files for our Node.js application. Then, we'll compile our JavaScript files into an executable file.
+
+### Table of Contents
+
+- [Key takeaways](#Key-takeaways "Key takeaways")
+- [Pre-requisites](#Pre-requisites "Pre-requisites")
+- [What are Executable (.exe) files](#What-are-Executable-(.exe)-files "What are Executable (.exe) files")
+- [Pros of executable files](#Pros-of-executable-files "Pros of executable files")
+- [Packages used](#Packages-used "Packages used")
+- [A simple web app](#A-simple-web-app "A simple web app")
+- [nexe](#nexe "nexe")
+    - [nexe installation](#nexe-installation "nexe installation")
+    - [Compile one javascript file into an exercutable using nexe](#Compile-one-javascript-file-into-an-exercutable-using-nexe "Compile one javascript file into an exercutable using nexe")
+    - [Compile your project into an executable using nexe](#Compile-your-project-into-an-exercutable-using-nexe "Compile your project into an exercutable using nexe")
+- [pkg](#pkg "pkg")
+    - [pkg installation](#pkg-installation "pkg installation")
+    - [Compile one javascript file into an executable using pkg](#Compile-one-javascript-file-into-an-exercutable-using-pkg "Compile one javascript file into an exercutable using pkg")
+    - [Compile your project into an executable using pkg](#Compile-your-project-into-an-exercutable-using-pkg "Compile your project into an exercutable using pkg")
+    - [File or directory not included Error](#File-or-directory-not-included-Error)
+- [Further practice](#Further-practice "Further practice")
+- [References](#References "References")
+
+### Key takeaways
+
+At the end of this tutorial, you would have learned the following:
+
+- What are `.exe` files and their importance.
+- Packages that we can use to compile your Node.js application into `.exe` files.
+- Creating a simple Node.js application.
+- Compiling Node.js files into an executable file for various operating systems.
+- Running the executable file in an operating system.
+
+### Pre-requisites
+
+For one to quickly follow up with this tutorial, you require a minimum of the following requirements:
+
+- Basic web development skills.
+- Some javascript coding skills.
+- Node.js installed in your machine. If not, download it from their [website](https://nodejs.org/en/download/).
+- An IDE. I highly recommend Visual Studio Code which is free and highly customizable. You can download it from [here](https://code.visualstudio.com/download).
+- A stable internet connection.
+
+If you are all set on the above requirements, let's look at some packages to accomplish our goal.
+
+### What are Executable (.exe) files
```

Comment:
You have to specify if we are only working with Windows-based executable files.

Reply:
Ok.
The main purpose of this article was intended for windows but the packages support a variety of other O/S so I thought it would be better to mention it as a bonus! So should I change it?

Reply:
Ok let me get into it

Reply:
Please check to see if it is as expected